airbyte-integrations/connectors/source-plaid/source_plaid/source.py

```diff
@@ -50,7 +61,9 @@ def read_records(
         stream_slice: Mapping[str, Any] = None,
         stream_state: Mapping[str, Any] = None,
     ) -> Iterable[Mapping[str, Any]]:
-        balance_response = self.client.accounts_balance_get(AccountsBalanceGetRequest(access_token=self.access_token))
+        options = AccountsBalanceGetRequestOptions()
+        options.min_last_updated_datetime = self.min_last_updated_datetime
```

Comment:
```suggestion
        options.min_last_updated_datetime = datetime.format(self.start_date, "YYYY-MMTHH:MM:SSZ")
```
You can apply this formating to correct datetime the endpoints is waiting for. So users don't need to know if they CapitalOne or not, is going to work anyway.